cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/SessionManager.java

```diff
@@ -25,6 +27,7 @@
 import org.apache.kafka.common.utils.Time;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import static com.linkedin.kafka.cruisecontrol.servlet.KafkaCruiseControlServletUtils.*;
```

Comment:
Let's avoid wildcard imports.

Reply:
Minor style comment: on a second look, I feel like we can completely avoid this static import using `KafkaCruiseControlServletUtils.endPoint()` in just a single line -- feel free to use an approach that you would prefer.